servers/src/main/java/tachyon/worker/block/TieredBlockStore.java

```diff
@@ -74,6 +74,12 @@
   private final Evictor mEvictor;
   private List<BlockStoreEventListener> mBlockStoreEventListeners =
       new ArrayList<BlockStoreEventListener>();
+  /** A list of pinned blocks fetched from the master */
+  private final Set<Integer> mPinnedInodes = new HashSet<Integer>();
+  /** A list of blocks that are currently being locked */
+  private final Set<Long> mLockedBlocks = new HashSet<Long>();
+  /** A narrowed view provided to evictors and allocators */
+  private BlockMetadataManagerView mMetadataView;
```

Comment:
Since in this version we don't keep a long-living `BlockMetadataManagerView`, it is unnecessary to keep `mMetadataView`